config/locales/en.yml

```diff
@@ -489,25 +489,15 @@ en:
       frequency:
         daily: daily, at around 3 pm
         weekly: weekly (Friday evening at around 6 pm)
-      header: Your email subscription has started
+      header:
+        create: Your email subscription has started
+        update: Your job alert has been updated
       next_step_details: 'We’ll email you details of any new teaching jobs that match the following criteria:'
       next_step: What happens next
-      title: Subscription confirmation
+      title:
+        create: Subscription confirmation
+        update: Subscription update
```

Comment:
I mildly prefer `Subscription update confirmation` as it quickly summarizes the content of the page for a screen-reader user

Reply:
I've Slacked Rose to ask if she considers titles when writing content